### ISSUE TITLE
Use aria-label for close button in image gallery

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/gmb.sk#420]
Close button in gallery modal is missing `aria-label`:

<img width="747" alt="Image" src="https://github.com/user-attachments/assets/48c5846e-fcef-404e-835a-8265dc54e506" />
